extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -124,14 +124,14 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
     try {
       const lookupResult = await this.view.pubLookup.lookupEmail(email);
       if (lookupResult && email) {
-        if (lookupResult.pubkey) {
-          const parsed = await opgp.key.readArmored(lookupResult.pubkey);
+        if (lookupResult.pubkey && lookupResult.pubkey.type === 'openpgp') {
+          const parsed = await opgp.key.readArmored(lookupResult.pubkey.unparsed);
           const key = parsed.keys[0];
           if (!key) {
             console.info('Dropping found but incompatible public key', { for: email, err: parsed.err ? ' * ' + parsed.err.join('\n * ') : undefined });
             lookupResult.pubkey = null; // tslint:disable-line:no-null-keyword
           } else if (! await PgpKey.usableForEncryption(lookupResult.pubkey) && ! await PgpKey.expired(key)) { // Not to skip expired keys
-            console.info('Dropping found+parsed key because getEncryptionKeyPacket===null', { for: email, fingerprint: await PgpKey.fingerprint(parsed.keys[0]) });
+            console.info('Dropping found+parsed key because getEncryptionKeyPacket===null', { for: email, fingerprint: parsed.keys[0].getFingerprint().toUpperCase() });
```

Comment:
Ideally getFingerprint would already be returning uppercased version, else for sure a similar bug will resurface. We always use uppercase everywhere, so it would be better to uppercase it in the common choke point.